cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -228,7 +228,7 @@
 default['travis_build_environment']['haskell_default_ghc'] = '7.10.3'
 default['travis_build_environment']['haskell_default_cabal'] = '1.22'
 
-gradle_version = '4.0.1'
+gradle_version = '4.5'
```

Comment:
Current latest version is 4.6
~~~~
gradle_version='4.6'
default['travis_build_environment']['gradle_version'] = gradle_version
default['travis_build_environment']['gradle_url'] = "https://services.gradle.org/distributions/gradle-#{gradle_version}-bin.zip"
 default['travis_build_environment']['gradle_checksum'] = '98bd5fd2b30e070517e03c51cbb32beee3e2ee1a84003a5a5d748996d4b1b915'
~~~~

Could you please update your configuration?